communication/moderation.md

```diff
@@ -53,9 +124,8 @@ Moderators _SHOULD_:
 The Kubernetes [Code of Conduct Committee] will have the final authority
 regarding escalated moderation matters. Violations of the Code of Conduct will
 be handled on a case by case basis. Depending on severity this can range up to
-and including removal of the person from the community, though this is
```

Comment:
Say something about guidance on removal in escalated situations outside of spammers will come from code of conduct committee.